cosmoz-grouped-list.js

```diff
@@ -388,19 +388,10 @@
 		 * @returns {Object|null} The group the item is in or null
 		 */
 		getItemGroup: function (item) {
-			if (!this._groupsMap) {
-				return;
-			}
-			var foundGroup;
-			this.data.some(function (group) {
-				var found = group.items.indexOf(item) > -1;
-				if (found) {
-					foundGroup = group;
-				}
-				return found;
-			});
-
-			return foundGroup;
+			return this._groupsMap && this.data.find(function (group) {
```

Comment:
The reason for the old implementation is that we haven't been able to move to ES6 yet.
Neither through transpiling or platform support adjustments.
Otherwise Array.find() would have been a given.
This implementation should have been an Array.filter().find() though in my opinion for readability.